tests/wpt/metadata/fetch/content-length/parsing.window.js.ini

```diff
@@ -74,8 +74,8 @@
   [Input: "Content-Length: 0x20". Expected: 42.]
     expected: FAIL
 
-  [Input: "Content-Length: 030\\r\\nContent-Length: 30". Expected: network error.]
-    expected: FAIL
+  [Input: "Content-Length: 030\\r\\nContent-Length: 30". Expected: 30.]
```

Comment:
`http` now uses an updated version of `itoa` to perform conversion: https://github.com/hyperium/http/commit/abe651273f4cf19cf9a247f376e9ece85becc722

Reply:
All of the tests now marked PASS can be removed from this file entirely.